src/main/java/org/jabref/gui/search/rules/describer/ContainsAndRegexBasedSearchRuleDescriber.java

```diff
@@ -40,6 +40,9 @@ public TextFlow getDescription() {
         }
 
         textList.add(getCaseSensitiveDescription());
+        textList.add(TooltipTextUtil.createText("\n\nHint: To search specific fields only, enter for example:\n"));
```

Comment:
Shouldn't this be localized?

Reply:
I translated the hint text - not the query itself. - Was the best thing, I could come up with.